src/isNodeSelectionForNodeType.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import {Selection, NodeSelection} from 'prosemirror-state';
+import {NodeType} from 'prosemirror-model';
+
+// Whether the seletion is a node for the node type provided.
+export default function isNodeSelectionForNodeType(
```

Comment:
this is currently only used once - is it likely we'll use it elsewhere?

Reply:
maybe later, I guess.